test/unit/notifications.server.test.ts

```diff
@@ -0,0 +1,33 @@
+import sendNotification from '../../app/lib/notifications.server';
+const { NOTIFICATIONS_EMAIL_USER } = process.env;
+describe('Notification server lib function test', () => {
+  const recipient: string = 'dev@starchart.invalid';
```

Comment:
tip: in TS, you only need to provide a type when it can't be inferred from the context.  I this case, it's clear that you're assigning a `string` so you can omit it.  It might seem odd to leave things out, but it's generally the way it's done.